components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/auth/dto/SCIMUser.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * <p>
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.apimgt.core.auth.dto;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This class contains the model of SCIM user
+ */
+public class SCIMUser {
+
+    private List<String> schemas;
+    private String userName;
```

Comment:
Can't do that because the model object needs to be tally with the real object. 
http://www.simplecloud.info/specs/draft-scim-core-schema-01.html#anchor3